cypress/integration/proposal_administration.ts

```diff
@@ -178,4 +178,216 @@ context('Proposal administration tests', () => {
       '.MuiTableSortLabel-active .MuiTableSortLabel-iconDirectionAsc'
     ).should('exist');
   });
+
+  const textQuestion = faker.random.words(3);
+  const dateQuestion = faker.random.words(3);
+  const boolQuestion = faker.random.words(3);
+  const multipleChoiceQuestion = faker.random.words(3);
+
+  let textQuestionId: string;
+  let dateQuestionId: string;
+  let boolQuestionId: string;
+  let multipleChoiceQuestionId: string;
+
+  it('Should be able to prepare proposal template', () => {
+    cy.login('officer');
+
+    cy.navigateToTemplatesSubmenu('Proposal templates');
+
+    cy.contains('default template')
+      .parent()
+      .get("[title='Edit']")
+      .first()
+      .click();
+
+    cy.createTopic('Topic for questions');
+
+    cy.get('[data-cy=show-more-button]')
+      .last()
+      .click();
+
+    cy.get('[data-cy=add-question-menu-item]')
+      .last()
+      .click();
+
+    cy.createBooleanQuestion(boolQuestion);
+    cy.contains(boolQuestion)
+      .closest('[data-cy=question-container]')
+      .find("[data-cy='proposal-question-id']")
+      .invoke('html')
+      .then(fieldId => {
+        boolQuestionId = fieldId;
+      });
+
+    cy.createDateQuestion(dateQuestion);
+    cy.contains(dateQuestion)
+      .closest('[data-cy=question-container]')
+      .find("[data-cy='proposal-question-id']")
+      .invoke('html')
+      .then(fieldId => {
+        dateQuestionId = fieldId;
+      });
+
+    cy.createMultipleChoiceQuestion(
+      multipleChoiceQuestion,
+      'One',
+      'Two',
+      'Three'
+    );
+    cy.contains(multipleChoiceQuestion)
+      .closest('[data-cy=question-container]')
+      .find("[data-cy='proposal-question-id']")
+      .invoke('html')
+      .then(fieldId => {
+        multipleChoiceQuestionId = fieldId;
+      });
+
+    cy.createTextQuestion(textQuestion, false, false);
+    cy.contains(textQuestion)
+      .closest('[data-cy=question-container]')
+      .find("[data-cy='proposal-question-id']")
+      .invoke('html')
+      .then(fieldId => {
+        textQuestionId = fieldId;
+      });
+  });
+
+  const proposalTitle = 'visi te tusee'; // faker.random.words(3);
+  const answerDate = '2030-01-01';
+  const answerMultipleChoice = 'One';
+  const answerText = 'kamert vins klusee'; // faker.random.words(3);
```

Comment:
Not sure, but maybe you forgot to use the faker version after testing?

Reply:
Thanks